Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
+MAINTAINER Alexandros Sigaras <als2076@med.cornell.edu>
+#===========================#
+# Clone cBioPortal			#
+#===========================#
+#RUN git clone https://github.com/cBioPortal/cbioportal.git; \
+ADD / /cbioportal
```

Comment:
Small note: the `ADD` command is disfavoured for simple cases like this in favour of the `COPY` command (https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy). In addition, I do not believe I’ve seen this use of absolute paths as source operands documented anywhere, does it do the same thing as this?
```Dockerfile
COPY . /cbioportal
```